shop/.env

```diff
@@ -0,0 +1,3 @@
+DOMAIN=192.168.56.102:8080
```

Comment:
To będzie działać tylko jak ma się komputer w tej konkretnej sieci? Czyli jak ja mam w 192.168.68.x, to nie odpalę sklepu? Może przejdzie localhost lub 127.0.0.1?

Reply:
Zrobiłem tak, że domyślnie jest localhost, oraz jest komentarz, który informuje o tym by to zmienić u siebie lokalnie w przypadku innego adresu